src/server/test/db/barchartReadingsTests.js

```diff
@@ -32,7 +32,7 @@ mocha.describe('Barchart Readings', () => {
 		// TODO: This is awkward. Is there a better way to have conn as a non-optional
 		// without requiring changing EVERY call to getBarchartReadings?
 		// If not, well, we need to change every call.
-		const barchartReadings = await Reading.getBarchartReadings([meter.id], moment.duration(2, 'h'), null, null, conn);
+		const barchartReadings = await Reading.getBarchartReadings([meter.id], moment.duration(2, 'h').toISOString(), null, null, conn);
```

Comment:
Not sure if need the value in string format or not.

Reply:
This was added to resolve issues after the upgrade of pg-promise. I don't know if this is the best fix but it works for now.